docs/source/contents/getting-started/index.md

```diff
@@ -46,4 +46,4 @@ docker-installation/index.md
 kubernetes-installation/index.md
 configuration/index.md
 cli.md
-```
\ No newline at end of file
+```
```

Comment:
![image](https://github.com/SeldonIO/seldon-core/assets/10928117/dc59a0be-c5b6-46b2-ad4c-8d3d57f930f3)

Reply:
Thanks @RafalSkolasinski, my `vim` has the "no new line at end of file" check on...

I've just redone the commit without...